docs/build/tutorials/platform/subnets/create-a-virtual-machine-vm.md

```diff
@@ -1466,6 +1464,39 @@ It's possible to give an alias for these IDs. For example, we can alias `Timesta
   ]
 }
 ```
+j
+### Installing a VM
+
+AvalancheGo searches for and registers plugins under the `plugins` directory of the [build directory](https://docs.avax.network/build/references/avalanchego-config-flags/#build-directory).
+
+To install the virtual machine onto your node, you need to move the built virtual machine binary under this directory. Virtual machine executable names must be either a full virtual machine ID (encoded in CB58), or a VM alias.
+
+1. Copy the binary into the plugins directory.
+```bash
+cp -n <path to your binary> $GOPATH/src/github.com/ava-labs/avalanchego/build/plugins/
+```
+2. Load the binary with the `loadVMs` API.
+```bash
+curl -sX POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"admin.loadVMs",
+    "params" :{}
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/admin | jq
```

Comment:
do we need to specify jq here? if it's possible can we omit using additional commands?

Reply:
No, just nice for formatting. Removed.